katello/katello-backup

```diff
@@ -7,14 +7,17 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
```

Comment:
@ehelms I guess your concern is having both mongodb and pulp listed here? Maybe we could call `pulp` `pulp-content` or something like that?